### PR TITLE
Added build script for drupal/terms_of_use

### PR DESCRIPTION
**Pull request by Sunidhi-Gaonkar1**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@prashantkhoje please review

File: d/drupal-terms_of_use/drupal-terms_of_use_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  echo "$CORE_PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"  
+
+fi
+
+if ! git clone $CORE_PACKAGE_URL $CORE_PACKAGE_NAME; then
+    	echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+		echo "$CORE_PACKAGE_URL $CORE_PACKAGE_NAME"
+        echo "$CORE_PACKAGE_NAME  |  $CORE_PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+    	exit 1
+fi
```
Comment (prashantkhoje) [inline d/drupal-terms_of_use/drupal-terms_of_use_ubi_8.5.sh:43]:
```suggestion
# Check if package exists
if [ -d "$CORE_PACKAGE_NAME" ] ; then
	rm -rf $CORE_PACKAGE_NAME
	echo "$CORE_PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package"
fi

if ! git clone $CORE_PACKAGE_URL $CORE_PACKAGE_NAME; then
	echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
	echo "$CORE_PACKAGE_URL $CORE_PACKAGE_NAME"
	echo "$CORE_PACKAGE_NAME  |  $CORE_PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
	exit 1
fi
```

File: d/drupal-terms_of_use/drupal-terms_of_use_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+cd $CORE_PACKAGE_NAME
+git checkout 8.9.0
+rm -rf composer.lock
+
+if ! composer install --no-interaction; then
+     	echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+	echo "$PACKAGE_URL $PACKAGE_NAME"
+	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails"
+	exit 1
+fi
```
Comment (prashantkhoje) [inline d/drupal-terms_of_use/drupal-terms_of_use_ubi_8.5.sh:54]:
```suggestion
if ! composer install --no-interaction; then
	echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
	echo "$PACKAGE_URL $PACKAGE_NAME"
	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails"
	exit 1
fi
```

File: d/drupal-terms_of_use/drupal-terms_of_use_ubi_8.5.sh [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  echo "$PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"  
+
+fi
+
+ if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+    	echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+		echo "$PACKAGE_URL $PACKAGE_NAME"
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+    	exit 1
+fi
```
Comment (prashantkhoje) [inline d/drupal-terms_of_use/drupal-terms_of_use_ubi_8.5.sh:71]:
```suggestion
# Check if package exists
if [ -d "$PACKAGE_NAME" ] ; then
	rm -rf $PACKAGE_NAME
	echo "$PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package"
fi

if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
	echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
	echo "$PACKAGE_URL $PACKAGE_NAME"
	echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
	exit 1
fi
```

File: d/drupal-terms_of_use/drupal-terms_of_use_ubi_8.5.sh [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_success_but_test_Fails"
+	exit 1
+else
+	echo "------------------$PACKAGE_NAME:install_&_test_both_success-------------------------"
+	echo "$PACKAGE_URL $PACKAGE_NAME"
+	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub  | Pass |  Both_Install_and_Test_Success"
+	exit 0
+fi
+
+#No test cases available.
```
Comment (prashantkhoje) [inline d/drupal-terms_of_use/drupal-terms_of_use_ubi_8.5.sh:91]:
```suggestion
echo "The test cases are not available."
```

**Review by prashantkhoje — CHANGES_REQUESTED** [review]

**Comment by Sunidhi-Gaonkar1** [comment]
@prashantkhoje Please review.

**Comment by prashantkhoje** [comment]
LGTM

**Pull request merged**